paragon/_overrides.scss

```diff
@@ -0,0 +1,149 @@
+      .main .nav-tab a {
+        &.active {
+          border-bottom-color: $main-nav-item-border-bottom-color !important;
+          border-bottom-style: solid !important;
+        }
+      }
+    }
+
+    @if variable-exists(main-nav-item-hover-border-bottom-color) {
+      .main .nav-tab a {
+        &:not(.active):hover {
+          border-bottom-color: $main-nav-item-hover-border-bottom-color !important;
+          border-bottom-style: solid !important;
+        }
+      }
+    }
+
+    @if variable-exists(user-dropdown-color) {
+      // Customize the color of the links in the user dropdown menu
+      .secondary .dropdown-user-menu .dropdown-item a {
+        color: $user-dropdown-color !important;
+      }
+    }
+  }
+
+  .main .nav-tab a,
+  .secondary .nav-item a:not(.btn):not(.help-link),
+  .secondary .nav-item .toggle-user-dropdown {
+    // Set the color of the dropdown arrow of the user dropdowm menu
+    color: $link-color !important;
+  }
+
+
+    @if variable-exists(customize-sign-in-btn) {
+      // Customize the sign-in button on the home page header
+      a.sign-in-btn {
+        @include btn-styles(
+          $btn-sign-in-bg,
+          $btn-sign-in-color,
+          $btn-sign-in-border-color,
+          $btn-sign-in-hover-bg,
+          $btn-sign-in-hover-color,
+          $btn-sign-in-hover-border-color
+        );
+      }
+
+    @if variable-exists(customize-register-btn) {
+      // Customize the register button on the home page header
+      a.register-btn {
+        @include btn-styles(
+          $btn-register-bg,
+          $btn-register-color,
+          $btn-register-border-color,
+          $btn-register-hover-bg,
+          $btn-register-hover-color,
+          $btn-register-hover-border-color
+        );
+      }
+    }
+  }
+}
+
+.login-register {
+  .form-field .field-link {
+    // Customize the color of the links in the login/registration form.
+    color: $link-color;
+  }
+
+  .action-primary {
+    @if variable-exists(customize-logistration-action-btn) {
+      // Customize the sign-in/registration button on the login page
+      &.login-button,
+      &.register-button,
+      & {
+        @include btn-styles(
+          $btn-logistration-bg,
+          $btn-logistration-color,
+          $btn-logistration-border-color,
+          $btn-logistration-hover-bg,
+          $btn-logistration-hover-color,
+          $btn-logistration-hover-border-color
+        );
+
+        box-shadow: none;
+      }
+    }
+  }
+}
+
+footer.footer {
+  background: $footer-bg; // Footer color
+  background-color: $footer-bg !important; // Footer color
+  box-shadow: none !important;
+
+  @if variable-exists(customize-footer-border) {
+    // Customize the footer border
+    border-top: $footer-border-top-width solid $footer-border-bottom-color !important;
+    border-bottom: $footer-border-bottom-width solid $footer-border-bottom-color;
+  }
```

Comment:
A small mixin could also reduce this.

Reply:
I haven't done that level of optimisation yet, since the aim was just to get some major UI theming elements in. 

Future tickets will refine, deduplicate, and clean up some of the code here.